patches/0073-libavfi-dnn-add-LibTorch-as-one-of-DNN-backend.patch

```diff
@@ -922,12 +922,12 @@ index 532b089002..cbaa1e601f 100644
              return AV_PIX_FMT_BGR24;
          }
 +    } else if (data->dt == DNN_FLOAT) {
++        //for libtorch dnn_processing only
 +        switch (data->order) {
 +        case DCO_RGB_PLANAR:
 +            return AV_PIX_FMT_GBRP;
 +        default:
-+            av_assert0(!"unsupported data pixel format.\n");
-+            return AV_PIX_FMT_GBRP;
++            return AV_PIX_FMT_NONE;
```

Comment:
Only modified code in here, remove unnecessary `av_assert0` here, which would cause regression for other backends inference.